indexed_vector.h

```diff
@@ -90,6 +90,14 @@ class IndexedVector {
     return std::move(elems);
   }
 
+  int size() {
```

Comment:
If you think these two are by any means not required, please tell me, I'll just use

`*indexed_vector_ref*.getElems().size()`

instead.